README.md

```diff
@@ -72,17 +72,20 @@ An _example_ of a valid `params.csv` file can be found in [`params.csv`](/src/pa
 The first row of the file (the headers) contains revocation model parameters **in fixed order**.
 
 - `State`
-- `Data Year`
-- `Revocations Population [C&C input]`
-- `Population Fraction [C&C input]`
-- `Revocations Admissions [C&C input]`
-- `Admissions Fraction [C&C input]`
+- `Revocations Population 2019 [C&C input]`
+- `Population Fraction 2019 [C&C input]`
+- `Revocations Admissions 2019 [C&C input]`
+- `Admissions Fraction 2019 [C&C input]`
+- `Revocations Population 2020 [C&C input] *If no data, enter same numbers as 2019*`
+- `Population Fraction 2020 [C&C input] *If no data, enter same numbers as 2019*`
 - `total cost per inmate [input]`
 - `marginal cost per inmate [input]`
 - `Number of facilities [input]`
-- `State-wide capacity [input]`
-- `Total Population`
-- `New Offense Population`
+- `state-wide capacity`
+- `Total Population 2019`
+- `Total Population 2020`
+- `New Offense Population 2019`
+- `New Offense Population 2020`
 - `Total Admissions`
 - `New Offense Admissions`
 - `New Offense Avg Time Served in Months`
```

Comment:
Commenting here because I can't comment further below: looking at the most recent `params.csv` file you shared over email, and the file in this repository, there's another 10 columns to the right of the last one listed here, `R A`, right?

`N Alpha_0 | R Alpha_0 | N Alpha_1 | R Alpha_1 | t_0 | t_1 | N P(t_1) | R P(t_1) | N P(t_1) baseline | R P(t_1) baseline`

Reply:
Hmm... You're right. I'd figured we only needed the user-input ones, but that was folly. The whole point of the rest of the spreadsheet is to compute the rest of the variables so the web app doesn't have to.

Editing...

Reply:
Okay, I looked back at these and actually most of the fields were for testing within the spreadsheet-based model. They're things for user inputs in the web model. (You can see on the [sheet itself](https://docs.google.com/spreadsheets/d/1BxuuieVPzvjJTpTtRwt9e1d_OJi6hQAvK45mBnSR0d8/edit#gid=991630608), columns X through AE are marked 'USER INPUT').

So I've added the only other couple of user-input / spreadsheet-generated to the docs here, and left the remainder (`N Alpha_1` and beyond) out.

Reply:
But are those fields still necessary for the application? If you take those fields out of the test file that's in the app, does it still function? If so, let's remove them from the test file as part of this PR so that we're not confusing to future developers.

Reply:
Confirmed via testing that they _are_ needed for the app to function, it does not load any of the states without them.

Reply:
Can you add them then, please? I don't think you have yet.

Reply:
Sorry, to clarify: the following two fields, which are new from the format previously and were added in this PR, _are_ necessary for the web app component:

> N Alpha_0
> R Alpha_0

The remaining fields from the spreadsheet are filler, and are _not_ needed.

> N Alpha_1 | R Alpha_1 | t_0 | t_1 | N P(t_1) | R P(t_1) | N P(t_1) baseline | R P(t_1) baseline

Reply:
If those are not needed, then please remove them from the test data file here in the app.

Reply:
Issued #96 to take care of this. I know it would've been cleaner to handle in this PR, but I don't see a way to do that from the gh web interface and don't have the repo synced down on my Linux virtualbox a/t/m.